File: call-management/call-queues/assign-multiple-call-queue-members/code-samples/assignMultipleCallQueueMembers.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
$accountId = '<ENTER VALUE>';
$groupId = '<ENTER VALUE>';

// POST BODY
$body = array(
    'addedExtensionIds' => array(
        '<ENTER VALUE>'  
    ),
    'removedExtensionIds' => array(
        '<ENTER VALUE>'  
    )
);

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->post("/restapi/v1.0/account/{$accountId}/call-queues/{$groupId}/bulk-assign", $body);
// PROCESS RESPONSE
?>

File: call-management/call-queues/assign-multiple-call-queue-members/code-samples/assignMultipleCallQueueMembers.py
# https://developers.ringcentral.com/my-account.html#/applications
# Find your credentials at the above url, set them as environment variables, or enter them below

# PATH PARAMETERS
accountId = '<ENTER VALUE>'
groupId = '<ENTER VALUE>'

# POST BODY
body = {
    'addedExtensionIds': [
        '<ENTER VALUE>'
    ],
    'removedExtensionIds': [
        '<ENTER VALUE>'
    ]
}

import os
from ringcentral import SDK
rcsdk = SDK(os.environ['clientId'], os.environ['clientSecret'], os.environ['serverURL'])
platform = rcsdk.platform()
platform.login(os.environ['username'], os.environ['extension'], os.environ['password'])
r = platform.post(f'/restapi/v1.0/account/{accountId}/call-queues/{groupId}/bulk-assign', body)
# PROCESS RESPONSE